package.json

```diff
@@ -24,12 +24,15 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
```

Comment:
Why do we need these 2 packages and the one below `@mui/material` for?

Reply:
Actually, I was working on a dark theme, yes but don't need it in this PR.